pkg/controller/deployment/util/deployment_util.go

```diff
@@ -362,27 +680,6 @@ func countAvailablePods(pods []api.Pod, minReadySeconds int32) int32 {
 	return availablePodCount
 }
 
-func IsPodAvailable(pod *api.Pod, minReadySeconds int32, now time.Time) bool {
```

Comment:
Why do you need to touch these since you are just moving the package? Same question for all deletions in this file.